reggie/ingestion/preprocessor/michigan_preprocessor.py

```diff
@@ -188,6 +187,11 @@ def fill_empty_columns(df):
                     },
                     inplace=True,
                 )
+            # This is new
+            elif ("IS_ABSENTEE_VOTER" not in hdf.columns) and (
+                "IS_PERMANENT_ABSENTEE_VOTER" not in hdf.columns
```

Comment:
(minor comments) Why are we checking both columns here?

Reply:
It's the opposite, we're checking that neither of those columns is there. I suppose, we could just check if the renamed column is not there at this point but we don't want to overwrite the column that exists, if it does with nans

Reply:
Oh also WV just needed a reupload, the zip was corrupted.